Dockerfile

```diff
@@ -10,6 +10,7 @@ ENV CI=true
 
 RUN chmod +x /entrypoint.sh && \
     chmod +x /generate_db_schema_documentation.py && \
-    npm install
+    npm install && \
+    pip3 install 'md-toc==7.0.1'
```

Comment:
Instead, please put this in a `requirements.txt` file and install that. Otherwise, this becomes unmaintainable long term.